src/olympia/yara/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django_extensions.db.fields.json import JSONField
+
+from olympia.amo.models import ModelBase
+from olympia.files.models import FileUpload
+
+
+class YaraResults(ModelBase):
```

Comment:
maybe you want this to be called just `YaraResult` to avoid having specify the plural

Reply:
I did that initially but it sounds weird.. Maybe I should pick a different name than `results` (although I like it).

Reply:
I switched to `YaraResult` 🤷‍♀